topics/icrc_3x_get_delegation.md

```diff
@@ -0,0 +1,74 @@
+# ICRC-3x: Get Delegation
```

Comment:
I think we agreed to move this file to a different PR?

Reply:
I'm okay with both. Keeping it in this one is easier as we don't lose functionality. And as it is declared as a draft, it doesn't hurt to keep it here. What do you think?

Reply:
Ok, fair enough. 👍